README.md

```diff
@@ -1,15 +1,50 @@
 # Volto Hydra (volto-hydra)
 
-A volto addon to let you edit content in realtime when you have many frontends written in any framework.
+A volto addon to turn Plone Volto into a decoupled editor for headless Plone.
 
-## Connecting Your Frontend with Volto-Hydra AdminUI
+Why Hydra? because it lets you have many frontends (heads) written in many framework all connected to the same body (plone) while still letting 
+you edit the content directly into your frontend. 
 
-This guide will help you set up Volto-Hydra (AdminUI) and connect your frontend to it. The process is divided into two main steps:
+It is a GSoC project to prove with the following goals
+- Provide an editing UI as similar to Volto as possible (with some Quanta design ideas)
+- Do so with the simplest integrator instructions possible
 
-1. Setting up Volto-Hydra (AdminUI)
-2. Setting up your frontend to connect with Volto-Hydra
+It's Volto without having to learn any Volto.
 
-### Step 1: Setting Up Volto-Hydra (AdminUI) 
+## Want to try the editor?
+
+You can try out the editing experience now by logging into https://hydra.pretagov.com and pasting in the url of one of the available 
+frontends already deployed for this demo site.
+
+The first frontend you can test with is ... TODO
+
+Note: not everything works yet. Follow the progress on the [Hydra Roadmap](https://github.com/orgs/collective/projects/3/views/4)
+or the [Hydra README](https://github.com/collective/volto-hydra)
+
+
+## Want to help? Make your own frontend
+
+You can build your own frontend in your favourite frontend framework, deploy it and then [submit a ticket to have it listed as
+one of the test frontends](https://github.com/collective/volto-hydra/issues).
+
+
+### Choose Your Framework
+
+- You can use any frontend framework (e.g., React, Next.js, Vue.js).
+- Fetch content from the Plone backend using the [@plone/client](https://github.com/plone/volto/tree/main/packages/client) 
+  library or the simple Fetch API.
+- You can start small without dynamic menus or complex blocks and work up to supporting more kinds of blocks as you go.
+
+TODO: link to more documentation on creating a frontend using @plone/client
```

Comment:
I think its better if we should have a seperate doc folder now. Adding different (*.md) files in that and refrencing them in the readme.md.

Reply:
I meant external docs if they are available..if they are not then it's out of scope for us to write docs on how to make a frontend.